tensorboard/backend/application_test.py

```diff
@@ -339,5 +339,23 @@ def testExceptions(self):
       application.TensorBoardWSGIApp(logdir, plugins, multiplexer, 0)
 
 
+class DbTest(tf.test.TestCase):
```

Comment:
We don't test that application properly proxies the database over to the TBPlugin context.